### PR TITLE
Updating some copy to be evergreen.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: buyers-guide-locales/en-US/share.properties [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 bg_sharing_facebook_body_2=Don’t buy a gift that spies 👀, or gets your friend hacked this holiday season! Use Mozilla’s Privacy Not Included buyer’s guide.
+bg_sharing_facebook_title_evergreen2=Check out *Privacy Not Included, Mozilla’s IoT Buyer’s Guide.
+# Feel free to keep, remove or replace the emoji.
+bg_sharing_facebook_body_evergreen2=Don’t buy tech that spies 👀, or gets your friend hacked! Read *Privacy Not Included, a buyer’s guide from Mozilla.
 
 # This string will be used as a tweet, keep the translation under 131 characters (including the variable). Feel free to rephrase as needed to fit.
 bg_sharing_twitter_1=Don’t buy a gift that spies👀 this holiday season. Privacy Not Included, a tech buyer’s guide by @Mozilla #PrivacyNI {twitterLink}
 # This string will be used as a tweet, keep the translation under 131 characters (including the variable). Feel free to rephrase as needed to fit.
 bg_sharing_twitter_2=Don’t buy a gift that hacks your friend 😩. Use @Mozilla’s Privacy Not Included buyer’s guide #PrivacyNI {twitterLink}
+# This string will be used as a tweet, keep the translation under 131 characters
```
Comment (TheoChevalier) [inline buyers-guide-locales/en-US/share.properties]: We can actually update the limit now :) We can go up to 270 characters here

File: buyers-guide-locales/en-US/share.properties [hunk 2]
```diff
@@ -1,14 +1,23 @@
 bg_sharing_facebook_title_1=Privacy Not Included: A Holiday Buyer’s Guide by Mozilla
 bg_sharing_facebook_body_1=This holiday season, learn which gadgets come with privacy included, using Mozilla’s buyer’s guide for connected gifts.
+bg_sharing_facebook_title_evergreen1=Privacy Not Included: An IoT Buyer’s Guide by Mozilla
```
Comment (alanmoo) [inline buyers-guide-locales/en-US/share.properties]: Where's this get used?

File: buyers-guide-locales/en-US/share.properties [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+bg_sharing_facebook_body_evergreen1=This year, learn what tech comes with privacy included, using Mozilla’s buyer’s guide for connected gadgets.
 
 bg_sharing_facebook_title_2=Check out Mozilla’s Holiday Buyer’s Guide
 # Feel free to keep, remove or replace the emoji.
 bg_sharing_facebook_body_2=Don’t buy a gift that spies 👀, or gets your friend hacked this holiday season! Use Mozilla’s Privacy Not Included buyer’s guide.
+bg_sharing_facebook_title_evergreen2=Check out *Privacy Not Included, Mozilla’s IoT Buyer’s Guide.
```
Comment (alanmoo) [inline buyers-guide-locales/en-US/share.properties]: Same question for the rest of the items changed here

File: buyers-guide-locales/en-US/share.properties (hunk 2 above)
Comment (ScottDowne) [reply buyers-guide-locales/en-US/share.properties]: IIRC a lot of these just go into share progress, and not used anywhere in the repo.

**Pull request merged**